contracts/test/v0.1/RequestResponse.config.ts

```diff
@@ -1,5 +1,5 @@
 export function requestResponseConfig() {
-  const maxGasLimit = 2_500_000
+  const maxGasLimit = 2_800_000
```

Comment:
I modified that to experiment the `gasLimit: maxGasLimit + 300_000`
I modified it to 2.800.000. which is exactly the same as 2.500.000 + 300.000, but it didnot run. The error return even more confusing: "EVM reverts without a reason."

Reply:
@KelvinThai We should be careful about that because this initial `maxGasLimit` is then used everywhere. As far as I know the 300K extra gas was used in one of the tests. Now the increased value has impact on every test.

Reply:
OK. let's keep the config 2_500_000 consistently

Reply:
and let's remove the predefine `gasLimit` in the call, it would run with default